common/next/next.config.js

```diff
@@ -40,12 +40,12 @@ const createConfig =
           if (phase === PHASE_DEVELOPMENT_SERVER) {
             return [
               {
-                source: '/api/users/me',
-                destination: `${identityHost}/api/users/me`,
+                source: '/account/api/users/me',
+                destination: `${identityHost}/account/api/users/me`,
```

Comment:
Think these might no longer be required?

Reply:
I think these are for local development when the identity app is on a different host to the content/catalogue apps, so I _think_ it will still be needed. I reckon we might want to do development across all apps with docker-compose including a reverse proxy or something, but that's a yak to shave later on...